ios/rüm/LastTaskCell.swift

```diff
@@ -33,18 +33,10 @@ class LastTaskCell: UITableViewCell {
         taskLabel.text = task
         if let imageUrl = NSURL(string: photo) {
             if let data = NSData(contentsOfURL: imageUrl) {
-                self.profileImageView.image = UIImage(data: data)
+                let image = UIImage(data: data)
+                self.profileImageView.image = image
+                self.backgroundImageView.image = image
```

Comment:
wait why are we loading the user's profile image into both the background and the cell? I liked it the other way with the background being the task, foreground imageview being the user